src/com/itschool/session4/homework/challenge02/UnaryOperators.java

```diff
@@ -0,0 +1,18 @@
+package com.itschool.session4.homework.challenge02;
+
+public class UnaryOperators {
+    public static void main(String[] args) {
+        int x = 1;
+
+        System.out.println(++x);
+
+        System.out.println(--x);
+
+        System.out.println(x++);
+
+
```

Comment:
one empty line is fine and best practice when dealing with stuff you want to separate a little bit, but more than one is not ok since the code is being pushed at the bottom of the file and gets harder to read.

Reply:
thx for observations